acceptance/tests/puppetserver_requests.rb

```diff
@@ -0,0 +1,26 @@
+require 'beaker-http'
+
+test_name 'Ensure that requests can be built out to the puppetserver'
+
+step 'install latest released puppet agent' do
+  install_puppet_agent_on(hosts)
+end
+
+step 'install the latest puppet-server on the master' do
+  install_package(master, 'puppetserver')
+  on master, 'service puppetserver start'
```

Comment:
This is necessary to test the class method `configure_private_key_and_cert_with_puppet`. But I agree with the sentiment that it feels like there should be a presuite that stands up a simple http server for the acceptance to actually hit.